store/tikv/client_batch.go

```diff
@@ -461,6 +463,7 @@ func removeCanceledRequests(entries []*batchCommandsEntry,
 
 func sendBatchRequest(
 	ctx context.Context,
+	c *rpcClient,
```

Comment:
I think it's wierd. `recycle` is `rpcClient`'s duty, not `batchConn`. So let `batchConn` send events to `rpcClient` is better to call `rpcClient`'s methods here.

Reply:
I need an `eventCh` here to send message to, hence `c.eventCh` and this `c *rpcClient`